setup.cfg

```diff
@@ -25,6 +25,6 @@ license = BSD
 url = http://astropy.org/
 edit_on_github = False
 github_project = astropy/astropy
+# version should be PEP386 compatible (http://www.python.org/dev/peps/pep-0386)
+version = 0.0.dev
 
-[entry_points]
-astropy-package-template-example = packagename.example_mod:main
```

Comment:
This should not be removed. Must be an artifact from rebasing. Please add it back in.